core/storage/user/gae_models.py

```diff
@@ -345,6 +345,21 @@ def get_by_normalized_username(cls, normalized_username):
         return cls.get_all().filter(
             cls.normalized_username == normalized_username).get()
 
+    @classmethod
+    def get_by_email(cls, email):
+        """Returns a user model given an email.
+
+        Args:
+            email: str. The user's email.
+
+        Returns:
+            UserSettingsModel | None. The UserSettingsModel instance which
+            contains the same email. If multiple users are present, log an error
```

Comment:
Is the "log an error" actually happening?

Reply:
No, removed it